### PR TITLE
[Backport][ipa-4-7] Fix issues found by pylint 2.2.0

### PR DESCRIPTION
**Pull request by tiran**
Manual backport of PR #2602 

* pylint 2.2: Fix unnecessary pass statement
* pylint: Fix duplicate-string-formatting-argument 
* pylint: also verify scripts 
* Address misc pylint issues in CLI scripts

See https://pagure.io/freeipa/issue/7772

### CODE REVIEW

**Comment by tiran** [comment]
re-run: ``curl error (28): Timeout was reached for https://mirrors.fedoraproject.org/metalink?repo=updates-released-f28&arch=x86_64``

File: install/tools/ipa-replica-manage.in [hunk 1]
```diff
@@ -720,15 +720,15 @@ def clean_dangling_ruvs(realm, host, options):
             # get_ruv_both_suffixes returns server names with :port
             # This needs needs to be split off
             if ruv_dict.get('domain'):
-                master_info['ruvs'] = set([
+                master_info['ruvs'] = {
                     (re.sub(':\d+', '', x), y)
```
[the remark this answers is not in the document]
Comment (tiran) [reply install/tools/ipa-replica-manage.in]: It's still a set, but it uses the set comprehension syntax that was introduced in Python 2.7:

```
>>> {1, 2, 3}
set([1, 2, 3])
>>> {i**2 for i in range(4)}
set([0, 1, 4, 9])
```

**Comment by tiran** [comment]
This PR has to land ASAP to unblock any and all backports to 4.7 branch.

**Comment by Tiboris** [comment]
ipa-4-7:

* bc3e24c1725a429422481655d72e80e0dad62456 pylint 2.2: Fix unnecessary pass statement
* 0ae08969c16ea51784517dec5dee8c54dc416a90 pylint: Fix duplicate-string-formatting-argument
* f991319da03fbdec442fe55a06086cf8485d642e pylint: also verify scripts
* fa78de9ae2db865dbe9f9f8f6a7a48472b73c32d Address misc pylint issues in CLI scripts

**Pull request closed without merging**